test/cctest/test_dotenv.cc

```diff
@@ -0,0 +1,535 @@
+﻿#include <string>
+#include <sstream>
+#include "gtest/gtest.h"
+#include "node_test_fixture.h"
+#include "cppnv/EnvReader.h"
+
+using cppnv::EnvPair;
+using cppnv::EnvReader;
+using cppnv::EnvStream;
+using std::string;
+
+
+class DotEnvTest : public EnvironmentTestFixture {
+};
+
+TEST_F(DotEnvTest, ReadDotEnvFile) {
+  // std::vector<EnvPair*> env_pairs;
```

Comment:
Can you remove commented out codes?

Reply:
will do and the other requests here after my day job.

Reply:
Removed all the commented out code